Assignment6.Rmd

```diff
@@ -48,22 +48,26 @@ post(c.tree1, file = "tree1.ps", title = "MOOC") #This creates a pdf image of th
 #If we are worried about overfitting we can remove nodes form our tree using the prune() command, setting cp to the CP value from the table that corresponds to the number of nodes we want the tree to terminate at. Let's set it to two nodes.
 
 ```{r}
-c.tree2 <- prune(c.tree1, cp = )#Set cp to the level at which you want the tree to end
+c.tree2 <- prune(c.tree1, cp = 0.01000)#Set cp to the level at which you want the tree to end
 
 #Visualize this tree and compare it to the one you generated earlier
 
-post(c.tree2, file = "tree2.ps", title = "MOOC") #This creates a pdf image of the tree
+post(c.tree2, file = "tree2.ps", title = "MOOC")#This creates a pdf image of the tree
+
 ```
 
 #Now use both the original tree and the pruned tree to make predictions about the the students in the second data set. Which tree has a lower error rate?
+The error rate on each tree is the same since the same number of students were divided into each branch. I guess we chose variables (forum.post + grade) that were effective in informing the model. The tables below show the same numbers for each value of the table.
```

Comment:
That doesn't seem right. A pruned tree would be very unlikely to perform the same as an unpruned tree. I think by including an unscaled version of `forum.posts` the tree has effectively ignored `grade` in constructing the tree. I would try again either by scaling forum posts so that all variables are on the same scale, or use the other two variables only.